build/build-image/cross/VERSION

```diff
@@ -1 +1 @@
-v1.8.3-2
+v1.9.0rc1-1
```

Comment:
all the other references are to `1.9rc1`, while this one is `1.9.0rc1`. is this intended, @luxas?